metrics_utility/automation_controller_billing/collector.py

```diff
@@ -11,24 +11,31 @@
 # from awx.main.models import Job
 # from awx.main.access import access_registry
 # from rest_framework.exceptions import PermissionDenied
-from metrics_utility.automation_controller_billing.package import Package
+from metrics_utility.automation_controller_billing.package.factory import Factory as PackageFactory
+
 from awx.main.utils import datetime_hook
 from awx.main.utils.pglock import advisory_lock
 
-logger = logging.getLogger('awx.main.analytics')
+logger = logging.getLogger('metrics_utility.collector')
```

Comment:
@Ladas I noticed this excludes the logging from external logging (like Splunk), because "metrics_utility" root isn't configurable through API/settings